godot-core/src/private.rs

```diff
@@ -261,19 +275,76 @@ pub(crate) fn has_error_print_level(level: u8) -> bool {
     ERROR_PRINT_LEVEL.load(atomic::Ordering::Relaxed) >= level
 }
 
+#[cfg(debug_assertions)]
+struct ScopedFunctionStack(Vec<*const dyn Fn() -> String>);
```

Comment:
I'm usually a fan of named fields, unless it's really just a drop-in newtype, as it makes the usage code slightly more readable:
```suggestion
#[cfg(debug_assertions)]
struct ScopedFunctionStack {
    functions: Vec<*const dyn Fn() -> String>
};
```